.buildkite/pipeline.yml

```diff
@@ -0,0 +1,107 @@
+definitions:
+  terraform_docker: &terraform_docker
+    docker#v5.11.0:
+      image: "hashicorp/terraform:1.13"
+      workdir: "/workdir"
+      entrypoint: "/bin/sh"
+  tflint: &tflint
+    docker#v5.11.0:
+      image: "ghcr.io/terraform-linters/tflint:latest"
+      workdir: "/workdir"
+      entrypoint: "/bin/sh"
+  tfsec: &tfsec
+    docker#v5.11.0:
+      image: "aquasec/tfsec:latest"
+      workdir: "/workdir"
+      entrypoint: "/bin/sh"
+  packer: &packer
+    docker#v5.11.0:
+      image: "hashicorp/packer:1.11"
+      workdir: "/workdir"
+      entrypoint: "/bin/sh"
+  markdownlint: &markdownlint
+    docker#v5.11.0:
+      image: "davidanson/markdownlint-cli2:latest"
+      workdir: "/workdir"
+      command: [".", "--config", ".markdownlint.json"]
+  shellcheck: &shellcheck
+    shellcheck#v1.4.0:
+      files: "**/*.sh"
+
+steps:
+  - group: ":terraform: Root Modules"
+    steps:
+      - label: ":terraform: Format Check"
+        key: terraform-fmt
+        if_changed:
+          - "*.tf"
+        command: .buildkite/scripts/format_check
+        plugins:
+          - *terraform_docker
+
+      - label: ":terraform: Validate Modules"
+        key: terraform-validate-modules
+        if_changed:
+          - "*.tf"
+          - "*.tfvars"
+        command: .buildkite/scripts/validate
+        plugins:
+          - *terraform_docker
+
+      - label: ":lint-roller: TFLint"
+        key: tflint
+        if_changed:
+          - "*.tf"
+        command: .buildkite/scripts/tflint
+        plugins:
+          - *tflint
+
+      - label: ":shield: TFSec Security Scan"
+        key: tfsec
+        if_changed:
+          - "*.tf"
+        command: .buildkite/scripts/tfsec
+        plugins:
+          - *tfsec
+
+  - group: ":packer: Packer Checks"
+    steps:
+      - label: ":packer: Format Check"
+        key: packer-fmt
+        if_changed:
+          - "packer/linux/buildkite-vm-image.pkr.hcl"
+        command: .buildkite/scripts/packer_format
+        plugins:
+          - *packer
+
+      - label: ":packer: Validate"
+        key: packer-validate
+        soft_fail: true
+        if_changed:
+          - "packer/linux/"
+        command: .buildkite/scripts/packer_validate && echo "⚠️ This soft fails due to strict arch naming rules."
```

Comment:
Can the naming be updated to meet Packer's convention, or we have a clash elsewhere if we went this route?